src/main/java/org/cascadebot/cascadebot/utils/votes/VoteButtonGroupBuilder.java

```diff
@@ -130,6 +136,13 @@ public VoteButtonGroupBuilder setPeriodicConsumer(BiConsumer<List<VoteResult>, M
      */
     public VoteButtonGroup build(long owner, long channelId, long guild) {
         VoteButtonGroup buttonGroup = new VoteButtonGroup(owner, channelId, guild, periodicConsumer, timer);
+        buttonGroup.setIsMusicSkip(isMusicSkip);
+        if (isMusicSkip) {
+            buttonGroup.setFinishConsumer(finishConsumer);
+            buttonGroup.setTimerRunTime(10);
+            buttonGroup.setMaxTimeRunTime(30);
+            buttonGroup.setTimerRunTimeSkipAddon(5);
```

Comment:
Having these in this method will cause this method to eventually be way to big. These methods should be called in each individual instance (so in this case on the skip command its self).